vtex/admin-ui
# feat(filters): enable csx on all composable filter components

#### What is the purpose of this pull request?
Use admin ui core functions and enable csx usage on all filter components.

#### How should this be manually tested?
Check the csx example under FiltersSearch on storybook (ps the searchbox doesnt change a lot because our search component doesnt)



#### Types of changes

#### How does this PR make you feel? [:link:](http://giphy.com/)
